caas/kubernetes/provider/resources/clusterrolebinding.go

```diff
@@ -90,62 +92,50 @@ func (rb *ClusterRoleBinding) Delete(ctx context.Context, client kubernetes.Inte
 	return nil
 }
 
+func shouldDelete(crb1, crb2 rbacv1.ClusterRoleBinding) bool {
```

Comment:
This could do with a comment. What does `shouldDelete` apply to? `crb1` or `crb2`? etc